src/utilities/object-mapper/base.ts

```diff
@@ -74,6 +74,8 @@ export abstract class BaseObjectMap<T> {
   }
 
   protected findBaseType<T extends GraphQLType | GraphQLField<any, any>>(type: T): GraphQLType {
+    // this 'in' isnt familiar syntax to me.
+    // can you explain what it does?
```

Comment:
Here's the javascript implementation and usage.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in

Here's a brief explainer for how it works in typescript

https://www.typescriptlang.org/docs/handbook/advanced-types.html#type-guards-and-differentiating-types

Essentially, in typescript, the 'in' operator can be used for type discrimination when working with unions. Specifically, in this case a 'type' may have either a 'type' or 'ofType' field but not both. Without discriminating the unions, typescript will throw an error if trying to access the property directly (```type.type``` or ```type.ofType``` will error in this case)